pact/concrete-policies/guard-policy/guard-policy-v1.pact

```diff
@@ -35,6 +35,12 @@
     true
   )
 
+  (defcap GUARD-UPDATED:bool (token-id:string guard-name:string guard:guard)
```

Comment:
Would there be any drawbacks to re-using `GUARDS` event?

Reply:
I see that the functions are written to update individual guards.. I suggest creating an additional function `update-guards`, which takes in the entire guard object, and using `GUARDS` event on all of the update functions, so that listening to `GUARDS` can discover the latest guards.

Reply:
with the `update-guards` function, if someone wanted to rotate to a new key on all of the guards, then they can make 1 update call instead of 5 update calls.